docs/registry/create/_index.md

```diff
@@ -58,7 +58,19 @@ This interface defines how your model's server responds to API requests.
 
 To ensure the client interface you create returns the expected results, use the appropriate client interface defined in <file>components/\<resource-name\>/client.go</file> or <file>services/\<resource-name\>/client.go</file> in the [Viam RDK](https://github.com/viamrobotics/rdk/blob/main/) as a reference.
 
-For example, the `base` component client is defined in the [<file>client.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/client.go) file.
+For example, the `base` component client is defined in the [<file>components/base/client.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/client.go) file.
+
+See [Valid APIs to implement in your model](#valid-apis-to-implement-in-your-model) for more information.
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+To create a new resource model, start by referencing the corresponding built-in resource's implementation in the [Viam C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk/tree/main/src/viam/sdk).
```

Comment:
Currently, lines 68-73 are basically the same as lines 101-103. This will change when I move on to DOCS - 1527 (point to base.X files not client.X files) after this, where one or the other section will be removed. Keeping here just to match other two languages in the interim. With DOCS - 1527 all three will use similar, shorter, non-replicated copy! 👍